plugin/execute_command.py

```diff
@@ -37,17 +37,21 @@ def run_async() -> None:
             def handle_response(response: Any) -> None:
                 assert command_name
                 if isinstance(response, Error):
-                    sublime.message_dialog("command {} failed. Reason: {}".format(command_name, str(response)))
+                    self.handle_error_async(response, command_name)
                     return
-                msg = "command {} completed".format(command_name)
-                if response:
-                    msg += "with response: {}".format(response)
-                window = self.view.window()
-                if window:
-                    window.status_message(msg)
+                self.handle_success_async(response, command_name)
 
             session.execute_command(params, progress=True).then(handle_response)
 
+    def handle_success_async(self, result: Any, command_name: str) -> None:
```

Comment:
Can you add a docblock here noting that plugins may override this? Otherwise it’s not clear to me that any plugin would just override this (and accidentally break in the future).